in_toto/in_toto_verify.py

```diff
@@ -55,8 +55,9 @@ def in_toto_verify(layout_path, layout_key_paths):
   """
   try:
     verifylib.in_toto_verify(layout_path, layout_key_paths)
+
```

Comment:
Just a personal convention I picked up recently. I add blank lines after control flow blocks even if it's compound statements.

Our [style guideline just suggests](https://github.com/secure-systems-lab/code-style-guidelines#blank-lines) to
> Use blank lines in functions to indicate logical sections and help to offset comments.